.pre-commit-config.yaml

```diff
@@ -55,11 +55,13 @@ repos:
   - id: check-xml
   - id: check-yaml
   - id: end-of-file-fixer
+    exclude: \.(c|cc|cxx|cpp|frag|glsl|h|hpp|hxx|ih|ispc|ipp|java|js|m|mm|proto|vert)$
```

Comment:
c/cpp/h are already taken care of by clang-format and JS files are fixed via eslint. But why exclude the other types?

Reply:
I have just copied the this line from the clang-format hook without much thinking. 
```
files: \.(c|cc|cxx|cpp|frag|glsl|h|hpp|hxx|ih|ispc|ipp|java|js|m|mm|proto|vert)$
```
Do we need to adjust both then?